src/OmniSharp/Api/ProjectSystem/OmnisharpController.CurrentProject.cs

```diff
@@ -0,0 +1,48 @@
+using Microsoft.AspNet.Mvc;
+using OmniSharp.AspNet5;
+using OmniSharp.Models;
+using OmniSharp.MSBuild;
+using System.Linq;
+
+namespace OmniSharp
+{
+    public class CurrentProjectController
+    {
+        private readonly AspNet5Context _aspnet5Context;
+        private readonly OmnisharpWorkspace _workspace;
+        private readonly MSBuildContext _msbuildContext;
+
+        public CurrentProjectController(AspNet5Context aspnet5Context, MSBuildContext msbuildContext, OmnisharpWorkspace workspace)
```

Comment:
Why isn't this on the same controller as the /projects endpoint? It should also be the same information.

Also don't parse the project name and assume the target framework. We have a structure for that.